django_test_migrations/sql.py

```diff
@@ -0,0 +1,106 @@
+from functools import partial
+from typing import Callable, Dict, List, Optional, Union
+
+from django.core.management.color import Style, no_style
+from django.db import DefaultConnectionProxy, connections, transaction
+from django.db.backends.base.base import BaseDatabaseWrapper
+
+_Connection = Union[DefaultConnectionProxy, BaseDatabaseWrapper]
+
+DJANGO_MIGRATIONS_TABLE_NAME = 'django_migrations'
+
+
+def drop_models_tables(
+    database_name: str,
+    style: Optional[Style] = None,
+) -> None:
+    """Drop all installed Django's models tables."""
+    style = style or no_style()
+    connection = connections[database_name]
+    tables = connection.introspection.django_table_names(
+        only_existing=True,
+        include_views=False,
+    )
+    sql_drop_tables = [
+        connection.SchemaEditorClass.sql_delete_table % {
```

Comment:
Is this operation safe in terms of sql-injection? Maybe there are safer methods to join sql vars?

Should we even care?

Reply:
It should be safe, because `sql_delete_table` is string format defined by database backend (no user input) and its `table` argument is table's name (not any users input again), so it should be save.
This implementation is based on https://github.com/django/django/blob/c226c6cb3209122b6732fd501e2994c408dc258e/django/db/backends/base/schema.py#L342

Reply:
👍